instrumentation/opentelemetry_tesla/lib/middleware/opentelemetry_tesla_middleware.ex

```diff
@@ -86,14 +91,14 @@ defmodule Tesla.Middleware.OpenTelemetry do
     url = Tesla.build_url(url, query)
     uri = URI.parse(url)
 
-    attrs = %{
-      "http.method": http_method(method),
-      "http.url": url,
-      "http.target": uri.path,
-      "http.host": uri.host,
-      "http.scheme": uri.scheme,
-      "http.status_code": status_code
-    }
+    attrs =
+      %{}
+      |> Map.put(Trace.http_method(), http_method(method))
+      |> Map.put(Trace.http_url(), url)
+      |> Map.put(Trace.http_target(), uri.path)
+      |> Map.put(Trace.net_host_name(), uri.host)
```

Comment:
I am not sure about this one, please double-check me here

Reply:
Looks right. Looks like there are some additional host attributes in the conventions but those can be added later if they are needed.